HISTORY.rst

```diff
@@ -6,7 +6,7 @@ History
 Pending release
 ---------------
 
-* New release notes here
+* Added optional support for NumPy random state.
```

Comment:
I'd phrase it like the previous similar history entries "Reset NumPy's random state too"